src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2948,7 +2952,7 @@ because xbuild doesn't support framework reference assemblies.
 		ManifestFile="$(IntermediateOutputPath)android\AndroidManifest.xml"
 		AdditionalArguments="$(AndroidApkSignerAdditionalArguments)"
 	/>
-	<Message Text="Signed android package '$(ApkFileSigned)'" />
+	<Message Text="@(ApkAbiFilesSigned->'Signed android package %(filename)%(extension)','%0A')" />
```

Comment:
Using `%0A` as a "joiner" shouldn't be necessary here. Instead, [MSBuild Task Batching](https://msdn.microsoft.com/en-us/library/ms171473.aspx) should be useful, e.g.

```xml
<Message Text="Signed android package %(ApkAbiFilesSigned.Identity)" />
```

Reply:
You should probably also keep the existing log message, since it would be needed when `$(AndroidUseApkSigner)=False`. Maybe adding a `Condition` for either case?

Reply:
Good catch, I've cleaned it up